RULES.md

```diff
@@ -71,14 +81,17 @@ Example Value: ```dedicated```
 Description: Checks that CloudTrail is enabled in all regions. Use this rule only in your home region 
 
```

Comment:
Not your change, but this description seems incorrect for the rule specified. This should read something like
"Description: Checks that the tenancy of the EC2 Instances in this account are of type specified in DesiredTenancy, and reports complaince on the EC2 instance